### PR TITLE
Tiny comment fix to groupWriteResponse: metadata no longer exists

### PR DESCRIPTION
**Pull request by kkaneda**

### CODE REVIEW

File: multiraft/storage.go [hunk 1]
```diff
@@ -99,8 +99,8 @@ func newWriteRequest() *writeRequest {
 }
 
 // groupWriteResponse represents the final state of a persistent group.
-// metadata may be nil and lastIndex and lastTerm may be -1 if the respective
-// state was not changed (which may be because there were no changes in the request
+// lastIndex and lastTerm may be -1 if the respective state was not
+// changed (which may be because there were no changes in the request
 // or due to an error)
```
Comment (tamird) [inline multiraft/storage.go]: add a period at the end

**Comment by tamird** [comment]
LGTM. You should add `[ci skip]` to the commit message.

File: multiraft/storage.go (hunk 1 above)
Comment (kkaneda) [inline multiraft/storage.go]: done.

**Comment by kkaneda** [comment]
Sure, added `[ci skip]`. Will do next time for comment/doc changes.

**Comment by bdarnell** [comment]
LGTM

**Pull request merged**